script/deploy-docs.sh

```diff
@@ -93,6 +78,8 @@ https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}
 EOM
 
 echo Pushing changes
+git config --unset 'http.https://github.com/.extraheader' || true
```

Comment:
This is needed because of the GitHub CI runner configuration when pushing via https and PATs.
The `|| true` is set here because of the `set -e` BASH options from above.

Reply:
Example: `streamlink-twitch-gui-wiki` -> `streamlink-twitch-gui.wiki`:
https://github.com/streamlink/streamlink-twitch-gui-wiki/commit/ab68051a597d1ebf2f60e120b3bff2dfcc0ea92d
https://github.com/streamlink/streamlink-twitch-gui-wiki/runs/6873416348?check_suite_focus=true#step:3:10